TERMSOFSERVICE.md

```diff
@@ -270,11 +295,23 @@ interest in any of our Properties or any content other than your data and your
 user's content. These ToS do not grant you any rights to use our trademarks,
 logos, domain names, or other brand features.
 
+10.6. You transfer the plain, unrestricted in terms of time, place and content,
```

Comment:
For the optional Error Code Context functionality, which is not enabled by default, where a few lines of your code are displayed that caused an error, we added this sentence to allow us to store, display and format this code in our user interface. This content is always protected by access controls to be visible only to you.